package.json

```diff
@@ -43,11 +43,11 @@
   },
   "scripts": {
     "dev": "npx tsc -p tsconfig.node.json && node src/build/copy-to-dist.js",
-    "make": "make -C ../../.. snarkyjs",
     "make:no-types": "npm run clean && make -C ../../.. snarkyjs_no_types",
     "wasm": "./src/bindings/scripts/update-wasm-and-types.sh",
     "bindings": "cd ../../.. && ./scripts/update-snarkyjs-bindings.sh && cd src/lib/snarkyjs",
     "build": "node src/build/copy-artifacts.js && rimraf ./dist/node && npm run dev && node src/build/buildNode.js",
+    "build:bindings": "./src/bindings/scripts/update-snarkyjs-bindings.sh",
```

Comment:
I like `build:bindings`! but the script it replaces is `bindings`, not `make`. I still would like to have (a version of) `make` to build from source without changing files in the git source tree

Reply:
Fair enough! I added different script for this purpose, lemme know if it works for this case!